package.json

```diff
@@ -6,50 +6,47 @@
   "contributors": [
     {
       "name": "Analytical Graphics, Inc., and Contributors",
-      "url": "https://github.com/AnalyticalGraphicsInc/OBJ2GLTF/graphs/contributors"
+      "url": "https://github.com/AnalyticalGraphicsInc/obj2gltf/graphs/contributors"
     }
   ],
   "keywords": [
     "obj",
     "gltf"
   ],
-  "homepage": "https://github.com/AnalyticalGraphicsInc/OBJ2GLTF",
+  "homepage": "https://github.com/AnalyticalGraphicsInc/obj2gltf",
   "repository": {
     "type": "git",
-    "url": "git@github.com:AnalyticalGraphicsInc/OBJ2GLTF.git"
+    "url": "git@github.com:AnalyticalGraphicsInc/obj2gltf.git"
   },
   "bugs": {
-    "url": "https://github.com/AnalyticalGraphicsInc/OBJ2GLTF/issues"
+    "url": "https://github.com/AnalyticalGraphicsInc/obj2gltf/issues"
   },
   "main": "index.js",
   "engines": {
     "node": ">=4.0.0"
   },
   "dependencies": {
-    "async": "^2.1.4",
     "bluebird": "^3.4.7",
-    "byline": "^5.0.0",
-    "cesium": "^1.29.0",
-    "fs-extra": "^1.0.0",
-    "gltf-pipeline": "^0.1.0-alpha9",
-    "yargs": "^6.6.0"
+    "cesium": "^1.31.0",
+    "event-stream": "^3.3.4",
+    "fs-extra": "^2.0.0",
+    "gltf-pipeline": "^0.1.0-alpha11",
+    "mime": "^1.3.4",
+    "pngjs": "^3.0.1",
+    "yargs": "^7.0.1"
   },
   "devDependencies": {
-    "coveralls": "^2.11.15",
```

Comment:
This was removed but coveralls is still used in travis.xml and the coveralls script below.